src/glum/_glm_cv.py

```diff
@@ -617,8 +623,8 @@ def _get_deviance(coef):
                 y_train,
                 w_train,
                 offset_train,
-                col_means,
-                col_stds,
+                self.col_means,
+                self.col_stds,
```

Comment:
```suggestion
                self.col_means_,
                self.col_stds_,
```

Another one :)

(Btw it's a bit annoying that the CI unit tests only show up in the source repo and not under the PR. I should look into why that's the case at some point.)